src/main/java/net/fabricmc/tinyremapper/mixin/util/Box.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.tinyremapper.mixin.util;
+
+import java.util.Objects;
+
+public class Box<T> {
```

Comment:
Think you can use atomic reference/boolean etc instead

Reply:
`AomticBoolean` cannot hold a null value. Which is need to indicate the default value.